### ISSUE TITLE
revisions

### ISSUE THREAD

**Issue by zenrabbit** [MarioZuliani/Greenhouse-Experiment#1]
rename factor to temp
did you change density
show interactions etc in one plot
maybe move all EDA to end of rmd file and have what you will present in paper first
are data published?

**Comment by zenrabbit** [comment]
write all data to one file?
or have to csv files total after you tidy
actual_temps.csv and perhaps germination.csv
then lets review data, tidy it up, publish, then look at stats?

**Comment by MarioZuliani** [comment]
1) Factor now renamed to temp in all areas of code.
2) Nope did not change the density of seeds. Kept it consistent with the paper we published.
3) Working on showing interactions in one plot. Facet_wrap by temp produces an unappealing figure but facet_wrap by species looks interesting.
4) Will move EDA to end once I find a solid figure to show for publication.
5) Everything on KNB https://knb.ecoinformatics.org/view/doi:10.5063/F1GQ6W6R
6) So for the data I summarized the temperature data to produce the mean and max temperature and then produced it as its own dataset, combined it with the final number of individuals that germinated from the experiment to produce the file "final".
